contrib/athens3/zetacored-localbuild/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.8'
+services:
+  zetachain_mainnet_rpc:
+    platform: linux/amd64
+    build:
+      context: ../../..
+      dockerfile: Dockerfile
+    environment:
+      DAEMON_HOME: "/root/.zetacored"
+      NETWORK: athens3
+      #RESTORE_TYPE=snapshot/snapshot-archive/statesync
+      RESTORE_TYPE: "snapshot"
+      SNAPSHOT_API: https://snapshots.zetachain.com
+      TRUST_HEIGHT_DIFFERENCE_STATE_SYNC: 40000
+      CHAIN_ID: "athens_7001-1"
+      VISOR_NAME: "cosmovisor"
+      DAEMON_NAME: "zetacored"
+      DAEMON_ALLOW_DOWNLOAD_BINARIES: "false"
+      DAEMON_RESTART_AFTER_UPGRADE: "true"
+      UNSAFE_SKIP_BACKUP: "true"
+      CLIENT_DAEMON_NAME: "zetaclientd"
+      CLIENT_DAEMON_ARGS: ""
+      CLIENT_SKIP_UPGRADE: "true"
+      CLIENT_START_PROCESS: "false"
```

Comment:
Are we still using these arguments? If not, can we remove them?

Reply:
Are we still using Zetavisor, or have we moved to using the regular cosmovisor binary?

Reply:
We don't use zetavisor so I think the CLIENT_ arguments can be removed like @kingpinXD suggested.